### ISSUE TITLE
validation is missing for OpenTab

### ISSUE THREAD

**Issue by Debashis9012** [getgauge/taiko#607]
### Expected behavior
 Without opening browser if we perform any actions below should be the error message:

 ✘ Error: Browser or page not initialized. Call `openBrowser()` before using this API, run `.trace` for more info. 

### Actual behavior
> openTab()
 ✘ Error: Cannot read property 'removeAllListeners' of undefined, run `.trace` for more info.

### Steps to reproduce
➜  ~ taiko

Version: 0.8.0 (Chromium:76.0.3803.0)
Type .api for help and .exit to quit

> openTab()
 ✘ Error: Cannot read property 'removeAllListeners' of undefined, run `.trace` for more info.
> closeTab()
 ✘ Error: connect ECONNREFUSED 127.0.0.1:9222, run `.trace` for more info.

### Versions
```
Version: 0.8.0 (Chromium: 76.0.3803.0) a466ef8a83b45a473353e89c25b4b132857f22fb
```

**NOTE:** Validation has to be done on all the missing apis.

**Comment by nehashri** [comment]
Other api's where validate is missing: switchTo(), emulateNetwork(), emulateDevice(), setViewPort(), openTab(), closeTab(), overridePermissions(), clearPermissionOverrides(), setLocation(), doubleClick(), rightClick(), dragAndDrop(), clear(), scrollTo(), tap(), evaluate()

**Comment by nehashri** [comment]
Fix should be available with 0a86cf2

**Comment by Debashis9012** [comment]
This issue has been verified and tested the validations for other apis.
Commit version: ```0a86cf2```
